README.md

```diff
@@ -51,6 +74,22 @@ Ahhoz hogy a fenti példában a data változó értéke ne undefined legyen és
 Azaz fel kell vennünk a programunkban az alábbi két utasítást:<br>
   app.use(express.json());<br>
   app.use(express.urlencoded({extended: false}));
+
+  Összegezve
```

Comment:
Amit írtál teljesen korrekt, de a body-parser modulra a jelenlegi express verzióban nincs szükség. Nagyobb verzió váltásnál szokott olyan előfordulni, hogy kiveszik az említett opciókat az express-ből és abban az esetben szokták az általad említett módon elérni a request body-t. Amit linkeltem 1órás express videót annak a korábbi verziójában is az általad említett megoldást használja Brad, mert akkor hiányzott a beépített megoldás még. Az összegzést esetleg módosítsd második opcióra.

Reply:
Javítom